[WIP] Updated README.md

Docmilo opened the pull request:
Updated the README.md file with instructions on how to play the game for the players. This address issue #3. Can you @ShanewUU please review the instructions and ensure that they may sense.

Docmilo committed:
Updated README.md 

Updated the README.md file with instructions on how to play the game.

ShanewUU commented on README.md:
```diff
@@ -2,12 +2,15 @@
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
+5. When completed, the rows will disappear.
 
```
We should tell the player how to pause the game
```suggestion
6. To pause the game, just press the space bar again. 
```

ShanewUU commented on README.md:
```diff
@@ -2,12 +2,15 @@
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
```
I think it would be a good idea to move the goal of the game to the beginning of the instructions.

ShanewUU reviewed (changes requested):
Hi @Docmilo I have finished reviewing the instructions and reccommended some changes for you to consider.

Docmilo committed:
Update README.md

Co-authored-by: Milo Mindbender <44030697+ShanewUU@users.noreply.github.com>

Docmilo committed:
Update README.md

Made the suggested changes to the instructions on how to play the game and clearly defined the goal of the game.

ShanewUU reviewed (approved):
The instructions look complete now. Thanks for the great work @Docmilo
